src/components/AddItem.js

```diff
@@ -1,24 +1,83 @@
 import { collection, addDoc } from 'firebase/firestore';
 import { db } from '../lib/firebase';
+import { useState } from 'react';
+// When merging to main with Issue #3, we can remove setUser below
+import { setUser, getUser } from '../storage-utils/storage-utils';
+// When merging to main with Issue #3, we can remove the import below
+import { getToken } from '@the-collab-lab/shopping-list-utils';
 
 export default function AddItem() {
-  const submitHandler = async () => {
+  const [itemName, setItemName] = useState('');
+  const [frequency, setFrequency] = useState(7);
```

Comment:
Though not an issue for this PR, consider how we might avoid 'magic numbers' ([reference](https://codetuts.dev/magic-numbers-in-programming/)) here by creating a constant/map/object/dict/class that gives this `7` number more value and expressiveness to the code and the developer reading it!